Api/Controllers/EventsController.cs

```diff
@@ -58,10 +58,10 @@ public ActionResult GetEvent(Guid id)
     [HttpGet]
     [Route("{id}/posts")]
     [SwaggerOperation("List event's posts")]
-    public ActionResult<Paged<PostDto>> GetEventPosts([FromQuery] Paging paging, [FromRoute] Guid id)
+    public ActionResult<Paged<ListPostDto>> GetEventPosts([FromQuery] Paging paging, [FromRoute] Guid id)
     {
         var @event = eventService.AsUser(User.GetGuid()).GetEvent(id);
-        return Paged<PostDto>.PageFrom(@event.Posts.Select(p => p.ToDto()), CreationDateComparer.Instance, paging);
+        return Paged<ListPostDto>.PageFrom(@event.Posts.OrderByDescending(p => p.CreationDate).Select(p => p.ToListPostDto()), DummyComparer.Instance, paging);
```

Comment:
Why use a dummy comparer, when you can make one that orders the way you want it to? Doesn't the existing Post CreationDateComparer order from newest to oldest?